client/blocks/login/index.jsx

```diff
@@ -70,6 +69,7 @@ class Login extends Component {
 		onSocialConnectStart: PropTypes.func,
 		onTwoFactorRequested: PropTypes.func,
 		signupUrl: PropTypes.string,
+		redirectTo: PropTypes.string,
```

Comment:
```suggestion
		redirectTo: PropTypes.string,
		isPartnerSignup: PropTypes.bool,
```

This old way of setting up prop types doesn't really do much in my experience but probably worth keeping it up to date